### DISCUSSION TITLE
Pogsag & FMS - Verschiedene Frequenzen

### DISCUSSION THREAD

**Discussion by Stefan1121** [Schrolli91/BOSWatch#528] [General]
Hallo zusammen.

Gibt es die Möglichkeit mit Boswatch Pogsag sowie FMS gleichzeitig auszuwerten?
Die Frequenzen sind hierbei unterschiedlich.

Pogsag werte ich aktuell mit einem SDR-Stick aus. FMS soll nun
auf dem Raspberry Pi ebenfalls ausgewertet werden.

Viele Grüße Stefan

**Comment by flothi** [comment]
Das geht mit zwei Sticks und zwei Instanzen Boswatch, über eine Instanz meines Wissens nach nicht
